go.mod

```diff
@@ -425,3 +425,6 @@ replace (
 //
 // Then, run `make modtidy` in this repository.
 // This ensures that go.mod and go.sum are up-to-date.
+
+// DO NOT MERGE THIS CHANGE
+replace github.com/dapr/components-contrib => github.com/cgillum/dapr-components-contrib v0.0.0-20230410173729-f4047d1e15e3
```

Comment:
I know we have a code comment there, but just commenting here explicitly, so that we remember to update it before merging this PR.

Reply:
This has been removed. We're now getting the latest from `dapr/components-contrib`.